catalog/app/containers/Bucket/viewModes.spec.ts

```diff
@@ -1,136 +1,164 @@
 /**
  * @jest-environment jsdom
```

Comment:
I think this is obsolete since you don't use window.fetch anymore

Reply:
hm maybe